pandas/_libs/tslibs/strptime.pyx

```diff
@@ -139,14 +139,13 @@ def array_strptime(object[:] values, object fmt,
                 if is_coerce:
                     iresult[i] = NPY_NAT
                     continue
-                raise ValueError("time data %r does not match "
-                                 "format %r (match)" % (val, fmt))
+                raise ValueError(f"time data {repr(val)} does not match "
+                                 f"format {repr(fmt)} (match)")
```

Comment:
can you use str instead of repr and add the single-quotes explicitly

Reply:
> can you use str instead of repr and add the single-quotes explicitly

I just added the single-quoates, no need for a ```str()```